### ISSUE TITLE
Pylint tells me to uppercase a useless line, instead of telling me it is a useless line

### ISSUE THREAD

**Issue by bcm-at-zama** [PyCQA/pylint#6323]
### Bug description

I have the bug.py file

```
"""Docstring"""
useless_line = 42
```

If I do  `pylint bug.py`, it tells me

```
************* Module bug
bug.py:2:0: C0103: Constant name "useless_line" doesn't conform to UPPER_CASE naming style (invalid-name)
```

I wonder if this is not wrong, I believe pylint tells me to do a wrong fix. I have the impression that pylint should raise https://pylint.pycqa.org/en/latest/messages/warning/pointless-statement.html

If I apply the fix proposed by pylint, I get a 

```
"""Docstring"""
USELESS_LINE = 42
```

which is accepted as valid by pylint, but it is a pity that useless_line wasn't removed in the first place.

### Configuration

```ini
pylint --version
pylint 2.13.0
astroid 2.11.2
Python 3.9.10 (main, Jan 15 2022, 11:48:00) 
[Clang 13.0.0 (clang-1300.0.29.3)]

sw_vers
ProductName:	macOS
ProductVersion:	11.6
BuildVersion:	20G165
```

### Command used

```shell
pylint bug.py
```

### Pylint output

```shell
************* Module bug
bug.py:2:0: C0103: Constant name "useless_line" doesn't conform to UPPER_CASE naming style (invalid-name)

--------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 10.00/10, -10.00)
```

### Expected behavior

************* Module bug
bug.py:2:0: W0104: "useless_line" ... (pointless-statement)

--------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 10.00/10, -10.00)

### Pylint version

```shell
pylint --version
pylint 2.13.0
astroid 2.11.2
Python 3.9.10 (main, Jan 15 2022, 11:48:00) 
[Clang 13.0.0 (clang-1300.0.29.3)]
```

### OS / Environment

macOS

ProductName:	macOS
ProductVersion:	11.6
BuildVersion:	20G165

**Comment by DudeNr33** [comment]
A lot of people like keeping their constants that are reused in the Code base in a separate module without much other logic. 
That's why raising ``pointless-statement`` would be seen as a false positive by a lot of people.

**Comment by bcm-at-zama** [comment]
If we have `USELESS_LINE = 42`, I understand that you don't want to raise an error. 
But if we have `useless_line = 42`, I would raise `pointless-statement` instead of `invalid-name`

**Comment by Pierre-Sassoulas** [comment]
pointless statement means really pointless, like 
```python
"useless-line"
```
Here we assign a value to something, it's not pointless. By the way we should also not assume the upper case naming style for such variable as a lot of projects declare some module level variable in snake case i.e. ``logger =  logging.getLogger(__file__)`` see https://github.com/PyCQA/pylint/issues/4606#issuecomment-866320596
